src/JoomlaBrowser.php

```diff
@@ -38,6 +38,38 @@ class JoomlaBrowser extends WebDriver
 		'language'
 	);
 
+	protected $locatorObject;
+
+	/**
+	 * Function to instantiate the Locator Class, In case of a custom Template,
+	 * path to the custom Template Locator could be passed in Acceptance.suite.yml file
+	 *
+	 * for example: If the Class is present at _support/Page/Acceptance folder, simple add a new Parameter in acceptance.suite.yml file
+	 *
+	 * locator class: 'Page\Acceptance\Bootstrap2TemplateLocators'
+	 *
+	 * Locator could be set to null like this
+	 *
+	 * locator class: null
+	 *
+	 * When set to null, Joomla Browser will use the custom Locators present inside Locators.php
+	 *
+	 * @return \Locators
+	 */
+	public function instantiateLocator()
+	{
+		if (!(isset($this->config['locator class'])) || is_null($this->config['locator class']))
+		{
+			$this->locatorObject = new \Locators();
```

Comment:
as they are static, there is no gain in doing new. personally i would even prefer consts for the locators, even faster